reference/forms/types/repeated.rst

```diff
@@ -114,6 +115,15 @@ be displayed when the two fields do not match each other.
 Field Options
 -------------
 
+compare
+~~~~~~~
+
+**type**: ``callable`` **default**: ``null``
+
+The callable receives two values to compare, and should return a boolean: true if the fields are considered equal, false otherwise. The callable also receives the currently compared key as a third value.
```

Comment:
Is it worth having "try", "false", and function like "password_hash" surrounded with back sticks?